opsworks_deploy_python/recipes/logging-permissions.rb

```diff
@@ -0,0 +1,12 @@
+require 'chef/log'
+Chef::Log.level = :debug
+script "set_permissions" do
+  interpreter "bash"
+  user "root"
+  cwd "/home/deploy"
+  code <<-EOC
+    echo "set permissions for logging"
+    echo "set permissions for logging" >> set-perms.txt
+    sudo find /var/lib/aws/opsworks/chef/ -name "*.log" | xargs sudo chmod 777
```

Comment:
`chmod 777` raises an eyebrow for me. Is it really necessary to use the Devil's Bitmask here, or is there a more targeted permission level that we could use?

Reply:
I agree, we should avoid making world-writeable files unless there's some hard constraint forcing us (and if there is, we should record it in a comment and/or on the ticket).  Here, how about we use 755 for directories and 644 for files?

Reply:
changed permissions to 644 on log files